runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldSwitcherPanel.java

```diff
@@ -149,38 +173,58 @@ private JPanel buildHeader()
 		JPanel header = new JPanel(new BorderLayout());
 		JPanel leftSide = new JPanel(new BorderLayout());
 
-		worldHeader = new WorldTableHeader("World", orderIndex == WorldOrder.WORLD, ascendingOrder);
+		worldHeader = new WorldTableHeader("World", orderIndex == WorldOrder.WORLD, ascendingOrder, () ->
+		{
+			plugin.refresh();
+		});
 		worldHeader.setPreferredSize(new Dimension(WORLD_COLUMN_WIDTH, 0));
 		worldHeader.addMouseListener(new MouseAdapter()
 		{
 			@Override
 			public void mousePressed(MouseEvent mouseEvent)
 			{
+				if (mouseEvent.getButton() == 3)
+				{
+					return;
+				}
 				ascendingOrder = orderIndex != WorldOrder.WORLD || !ascendingOrder;
 				orderBy(WorldOrder.WORLD);
 			}
 		});
 
-		playersHeader = new WorldTableHeader("#", orderIndex == WorldOrder.PLAYERS, ascendingOrder);
+		playersHeader = new WorldTableHeader("#", orderIndex == WorldOrder.PLAYERS, ascendingOrder, () ->
+		{

```

Comment:
plugin::refresh

Reply:
What's the difference?

Reply:
its shorter. No reason to create additional runnable around running other "runnable" when you can reference it directly

Reply:
Gotcha, I tried it out in the code and realized why, forgot to delete the comment.